plenoptic/simulate/canonical_computations/steerable_pyramid_freq.py

```diff
@@ -777,14 +764,16 @@ def _recon_levels(self, pyr_coeffs, recon_keys, scale):
 
 
 
-    def steer_coeffs(self, angles, even_phase=True):
+    def steer_coeffs(self, pyr_coeffs, angles, even_phase=True):
```

Comment:
minor point: can this be a class method? only use of `self` is in checking `self.num_orientation` and `self.num_scales`, but both can be gotten from the keys of `pyr_coeffs`